repo/packages/R/riak/1/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "2.0",
+  "name": "riak",
+  "version": "2.0.0-rc2",
+  "framework": true,
+  "tags": ["basho", "framework", "database", "riak", "riak-kv", "riak-ts"],
+  "maintainer": "help@basho.com",
+  "description": "A distributed NoSQL key-value data store that offers high availability, fault tolerance, operational simplicity, and scalability.",
+  "scm": "https://github.com/basho-labs/riak-mesos.git",
+  "website": "http://basho-labs.github.io/riak-mesos/",
+  "postInstallNotes": "Thank you for installing Riak on Mesos. Visit https://github.com/basho-labs/riak-mesos for usage information.",
+  "preInstallNotes": "The Riak framework should have at least 1GB of RAM and 0.5 CPUs to perform successfully.",
```

Comment:
I just added `Documentation: https://github.com/dcos/examples/tree/master/1.8/riak` to the end cause there should be not only `Installation Documentation` but general documentation with examples.